homeassistant/components/sleepiq/strings.json

```diff
@@ -33,6 +33,17 @@
           "medium": "Medium",
           "high": "High"
         }
+      },
+      "core_temps": {
+        "state": {
+          "off": "Off",
+          "heating_push_low": "Heating Low",
+          "heating_push_med": "Heating Medium",
+          "heating_push_high": "Heating High",
+          "cooling_pull_low": "Cooling Low",
+          "cooling_pull_med": "Cooling Medium",
+          "cooling_pull_high": "Cooling High"
```

Comment:
why does this have push and pull inthere?

Please use sentence case

Reply:
Regarding sentence case: done

"pull" and "push" in the names correspond to the enums used in the `asyncsleepiq` library. See https://github.com/kbickar/asyncsleepiq/blob/ee304a436322e232072ce13d5a94f8e5743a5b0e/asyncsleepiq/consts.py#L24

Reply:
"push" and "pull" removed. Enum mapping implemented in https://github.com/home-assistant/core/pull/134718/commits/61adf38723b2dc5e8d29e6b4261fb5c994a635cf at @emontnemery's suggestion.